devdoc/CodeReviews.md

```diff
@@ -0,0 +1,55 @@
+Some general guidelines for code reviews.
+
+Code reviews have a few different goals:
+* Catch architectural or design problems
+* Catch bugs early (incorrect behaviour, inefficiencies, security issues)
+* Ensure the code is readable and maintainable.
```

Comment:
Additional item:
* Ensure the code follows the team's coding standards as set out in the document <link>

Reply:
done